Bot.py

```diff
@@ -345,6 +354,22 @@ async def get_pre(bot,message):
 ctx = discordjs.ext.commands.Bot(command_prefix = get_pre, case_insensitive = True)
 ctx.remove_command('help')
 
+ctx.add_check(parmision_cheque)
+
+
+
+ctx.parmissins = {
+    671777334906454026: parmissins["owner"]
```

Comment:
I cannot agree that the owner id is 671777334906454026.
```suggestion
    678401615333556277: parmissins["owner"]
```